scss/objects/_list-ui.scss

```diff
@@ -7,12 +7,12 @@
 $DBlist-ui-namespace: "o-list-ui" !default;
 
 .#{$DBlist-ui-namespace} {
-  margin: 0;
-  padding: 0;
-  list-style: none;
+    margin: 0;
+    padding: 0;
+    list-style: none;
 
-  &--dividers li + li {
-    border-top: 1px solid;
-    border-color: color(gray, x-dark, 0.1);
-  }
+    &--dividers li + li {
+        border-top: 1px solid;
+        border-color: color(gray, x-dark, 0.1);
```

Comment:
Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.